### PR TITLE
Tweak the concept of "draw" in 2.5.8 understanding for the spacing checks

### PR DESCRIPTION
**Pull request by patrickhlauke**
Admittedly, not sure this is absolutely necessary, but hopefully this allays concerns raised in the original issue.

Closes https://github.com/w3c/wcag/issues/3895

### CODE REVIEW

**Review by bruce-usab — CHANGES_REQUESTED** [review]
I like all the edits except first one, "possible" to "theoretically possible".

Maybe "conceptually possible" or leave as-is?  With the other edits, I do not think "draw" will be taken as a concrete literal requirement.

File: understanding/22/target-size-minimum.html [hunk 1]
```diff
@@ -81,7 +81,7 @@ <h3>Spacing</h3>
...
+            <p>When a target is smaller than 24 by 24 CSS pixels, it is <em>undersized</em>. In this case, we check if it at least has sufficient <em>spacing</em> by drawwing an imaginary 24 CSS pixel diameter circle over the undersized target, centered on the target's <a>bounding box</a>. For rectangular targets, the bounding box coincides with the target itself – thus, the circle is placed on the center of the target. If the target is <em>not</em> rectangular – for instance, if the target is clipped, has rounded corners, or if it's a more complex clickable SVG shape – we need to first determine the bounding box, and then find the box's center. Note that for concave shapes, the center of the bounding box may be outside of the target itself. Now, we center the circle on the center of the bounding box.</p>
```
Comment (patrickhlauke) [inline understanding/22/target-size-minimum.html:84]:
```suggestion
            <p>When a target is smaller than 24 by 24 CSS pixels, it is <em>undersized</em>. In this case, we check if it at least has sufficient <em>spacing</em> by drawing an imaginary 24 CSS pixel diameter circle over the undersized target, centered on the target's <a>bounding box</a>. For rectangular targets, the bounding box coincides with the target itself – thus, the circle is placed on the center of the target. If the target is <em>not</em> rectangular – for instance, if the target is clipped, has rounded corners, or if it's a more complex clickable SVG shape – we need to first determine the bounding box, and then find the box's center. Note that for concave shapes, the center of the bounding box may be outside of the target itself. Now, we center the circle on the center of the bounding box.</p>
```

**Comment by bruce-usab** [comment]
Discussed 9/13, minor change on call.  Move to Ready for approval.

File: understanding/22/target-size-minimum.html [hunk 2]
```diff
@@ -40,7 +40,7 @@ <h3>Exceptions</h3>
         <section id="size-requirement">
             <h3>Size requirement</h3>
 
-            <p>For a target to be "at least 24 by 24 CSS pixels", it must be possible to draw a solid 24 by 24 CSS pixel square, aligned to the horizontal and vertical axis such that the square is completely within the target (does not extend outside the target's area).</p>
+            <p>For a target to be "at least 24 by 24 CSS pixels", it must be theoretically possible to draw a solid 24 by 24 CSS pixel square, aligned to the horizontal and vertical axis such that the square is completely within the target (does not extend outside the target's area).</p>
```
Comment (patrickhlauke) [inline understanding/22/target-size-minimum.html:43]:
```suggestion
            <p>For a target to be "at least 24 by 24 CSS pixels", it must be conceptually possible to draw a solid 24 by 24 CSS pixel square, aligned to the horizontal and vertical axis such that the square is completely within the target (does not extend outside the target's area).</p>
```

**Review by bruce-usab — APPROVED** [review]

**Comment by mbgower** [comment]
This was date-stamped as being sent for WG approval on oct 10. I have left that stamp in place, but noting here that it was sent as part of the Nov 20 changes to the working group.

**Pull request merged**